.ruby-gemset

```diff
@@ -0,0 +1 @@
+my_app
```

Comment:
This file `.ruby-gemset` is for the ruby configuration, you can remove it since it is only needed to commit the implemented introspector.